.travis.yml

```diff
@@ -150,7 +150,10 @@ script:
   - pip install $(ls ./dist/*.whl)"[alldeps]"
   - bash .ci/tests.sh
   # Ensure bear requirements are in sync with the bear PipRequirement
-  - .ci/generate_bear_requirements.py
+  - .ci/generate_bear_requirements.py --check --update
+  - git clone https://github.com/moremoban/setupmobans ../setupmobans
```

Comment:
these three lines were moved to a separate job

Reply:
Fixed!